### ISSUE TITLE
for the tiny yolov3 3 layers file do we need to do the same as (./darknet partial cfg/yolov3-tiny.cfg yolov3-tiny.weights yolov3-tiny.conv.15 15) to get the pretrained conv file

### ISSUE THREAD

**Issue by ReynardTing** [AlexeyAB/darknet#3268]

**Comment by ReynardTing** [comment]
![Screenshot from 2019-05-29 13-00-20](https://user-images.githubusercontent.com/51064819/58530554-affa9280-8211-11e9-8ed0-2461017a5760.png)
![Screenshot from 2019-05-29 13-00-32](https://user-images.githubusercontent.com/51064819/58530555-affa9280-8211-11e9-93f2-fa74274930de.png)
here are some pic for reference please ask if you dont understand my stated question above thanks:)

**Comment by ReynardTing** [comment]
so as seen above i have downloaded the cfg file for tiny yolov3_3l. but do i need to partial it like de yolov3.tiny.conv.15(./darknet partial cfg/yolov3-tiny.cfg yolov3-tiny.weights yolov3-tiny.conv.15 15) and if so how do i go abt it?

**Comment by AlexeyAB** [comment]
Yes, use the same partial command and the same ` yolov3-tiny.conv.15` file

**Comment by ReynardTing** [comment]
meaning i will need to type (./darknet partial cfg/yolov3-tiny.cfg yolov3-tiny.weight yolov3-tiny_3l.conv.15 15)??

**Comment by ReynardTing** [comment]
sry as i am a bit new to this?

**Comment by ReynardTing** [comment]
so i have downloaded the 3 layers 3cfg weights from your github as shown below)
![Screenshot from 2019-05-30 09-37-49](https://user-images.githubusercontent.com/51064819/58602195-8b102900-82be-11e9-847c-4c2136af8d95.png)
so to get the weights file for 3 layes do i type this 
(./darknet partial cfg/yolov3-tiny.cfg yolov3-tiny.weights yolov3-tiny_3l.conv.15 15)
                                                                  or 
(./darknet partial cfg/yolov3-tiny.cfg yolov3-tiny.conv.15 yolov3-tiny_3l.conv.15 15)

sry as i am a bit new to this thanks for your help:)

**Comment by AlexeyAB** [comment]
Use this `./darknet partial cfg/yolov3-tiny.cfg yolov3-tiny.weights yolov3-tiny_3l.conv.15 15`

**Comment by ReynardTing** [comment]
and does this work if i use it on pjreddie Darknet git repository

> git clone https://github.com/pjreddie/darknet.git
instead of urs??
thanks

**Comment by AlexeyAB** [comment]
Yes

**Comment by ReynardTing** [comment]
so for this to start training i will use yolov3-tiny_3l.conv.15 as weights and yolov3-tiny_3l.cfg as my cfg file?
